bot/handlers/administration/subscription_status_handler.py

```diff
@@ -22,6 +24,9 @@ class SubscriptionStatusHandler(BotMessageHandler):
     def get_commands(self) -> List[str]:
         return ["subskrypcja", "subscription", "sub"]
 
+    def _get_validator_functions(self) -> List[Callable[[Message], Awaitable[bool]]]:
```

Comment:
zrób se może type alias w bazowej klasie na to convoluted gówno zwracane, typu `ValidatorFunctions`

https://stackoverflow.com/a/33045252

Reply:
ale technologia nie wiedziałem o tym